src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -346,15 +356,15 @@
        Protobuf_ExpectedOutputs with all possible output. An option is to include
        all existing outputs using Include with a wildcard, if you know where to look.
 
-       Note this is like Protobuf_PrepareCompile, but uses @(Protobuf) regardless
+       Note this is like Protobuf_PrepareCompile, but uses @(Protobuf_Rooted) regardless
        of the Compile metadata, to remove all possible outputs. Plugins should err
        on the side of overextending the Protobuf_ExpectedOutputs here.
 
        All ExpectedOutputs will be removed. -->
-  <Target Name="Protobuf_PrepareClean" Condition=" '@(Protobuf)' != '' ">
+  <Target Name="Protobuf_PrepareClean" Condition=" '@(Protobuf_Rooted)' != '' ">
```

Comment:
Same here, I think it would make sense for Protobuf_PrepareClean to depend on _Protobuf_SetProtoRoot, especially since this target isn't prefixed with _ which generally signals that it could be called independently.

Reply:
Good point! I will change it.

Reply:
It seems that `Condition` is checked before `DependsOnTargets`. So adding it to `Protobuf_PrepareClean` currently didn't work as it will skip it. So we would need split `Protobuf_PrepareClean` into two prefixed tasks that are not called directly.

Reply:
Are you still going to address this?

Reply:
See https://github.com/grpc/grpc/pull/22895#issuecomment-634482536

We have two options how to fix it but both have downsides and would change either the current behavior or would by asymmetrical to how creation is currently done which feels a bit strange. Would be good to @JunTaoLuo‘s opinion on the two suggestions. (Or three, since keeping it this way is also one.)

Reply:
Let's leave as is for now and if needed, we can address in a followup PR.